resources/views/auth/admission/finalize.blade.php

```diff
@@ -0,0 +1,80 @@
+@extends('layouts.app')
+@section('title')
+    <a href="{{ route('admission.applicants.index') }}" class="breadcrumb" style="cursor: pointer">Felvételi
+        jelentkezők</a>
+    <a href="#!" class="breadcrumb">Felvételi véglegesítés</a>
+@endsection
+
+@section('content')
+    <div class="card" style="margin-top:20px">
+        <div class="card-content">
+            <div class="row">
+                <div class="col">
+                    Hogyha a felvételi eljárás befejeződött, akkor a felvett jelentkezőket itt lehet
+                    jóváhagyni. A felvettek státusza a {{ $semester->tag }} szemeszterre automatikusan aktív lesz.
```

Comment:
This throws an exception if the periodic event is not yet set.

Also, for me, this is the spring semester and not the autumn one as it should be.

Reply:
The semester depends on the periodic event. You should set it for the autumn one. In the live system it looks fine.

Reply:
Oh, I see. Sorry then.